packages/battery/ios/battery.podspec

```diff
@@ -0,0 +1,20 @@
+#
```

Comment:
That's news to me :). Is that documented somewhere? There should probably be a comment on top of this file telling people to just ignore its values.